commodore/cluster.py

```diff
@@ -20,20 +20,26 @@ def fetch_cluster(cfg, clusterid):
 
 
 def reconstruct_api_response(target_yml):
-    target_data = yaml_load(target_yml)['parameters']
+    target_data = yaml_load(target_yml)
+    target_parameters = target_data['parameters']
+    target_classes = target_data['classes']
     api_resp = {
-        'id': target_data['cluster']['name'],
+        'id': target_parameters['cluster']['name'],
         'facts': {
-            'cloud': target_data['cloud']['provider'],
-            'distribution': target_data['cluster']['dist'],
+            'cloud': target_parameters['cloud']['provider'],
+            'distribution': target_parameters['cluster']['dist'],
         },
         'gitRepo': {
-            'url': target_data['cluster']['catalog_url'],
+            'url': target_parameters['cluster']['catalog_url'],
         },
-        'tenant': target_data['customer']['name'],
+        'tenant': target_parameters['customer']['name'],
     }
-    if 'region' in target_data['cloud']:
-        api_resp['facts']['region'] = target_data['cloud']['region']
+    if 'region' in target_parameters['cloud']:
+        api_resp['facts']['region'] = target_parameters['cloud']['region']
+    for cl in target_classes:
```

Comment:
So this is not an actual fact but gets extracted from the classes set. Why is this the case?

Reply:
That's only done in `--local` mode, where no actual call to the API is made but the existing local file is used and the according API response is generated.